admin_manual/file_workflows/retention.rst

```diff
@@ -19,6 +19,18 @@ The rule from the example will delete all files tagged with ``Temporary file`` 1
 You can also use the "Notify owner a day before a file is automatically deleted" option to
 make sure the file owner will get a notification before a file will be deleted.
 
+File age
+--------
+
+There are 2 options available that can be used to decide when to delete a file:
+
+- **Creation:** Time the file was created on the Nextcloud Server or uploaded to it.
+- **Last modification:** Time when the file was last modified.
+
+Since it often resulted in confusion in the past, the upload time is also considered
+in the modification case. Otherwise users would copy a file into the Nextcloud Sync client directory
+and the file would be deleted during the next background job execution.
```

Comment:
This still sounds a bit confusing to me. How it is "considered", why the file would be deleted.

What about something like:

> With the **Last modification** option, the uploading also counts as a modification.
> Thus uploading a long-modified file doesn't retire quickly after the upload.